tests/transforms/test_decompositions.py

```diff
@@ -72,9 +78,7 @@ def test_zyz_decomposition_torch(self, U, expected_gate, expected_params):
         assert len(obtained_gates) == 1
         assert isinstance(obtained_gates[0], expected_gate)
         assert obtained_gates[0].wires == Wires("a")
-        assert qml.math.allclose(
-            [x.detach() for x in obtained_gates[0].parameters], expected_params
-        )
+        assert qml.math.allclose(qml.math.unwrap(obtained_gates[0].parameters), expected_params)
```

Comment:
The original version caused errors when running locally, so I just tidied up a few things here.